src/Orckestra.Composer.Cart/Repositories/CartRepository.cs

```diff
@@ -542,17 +568,17 @@ public virtual Task<Overture.ServiceModel.Orders.Order> CompleteCheckoutAsync(Co
 
         public virtual async Task<List<ProcessedCart>> GetRecurringCartsAsync(GetRecurringOrderCartsViewModelParam param)
         {
-            if (param == null) { throw new ArgumentNullException(nameof(param)); }
-            if (param.CultureInfo == null) { throw new ArgumentException(GetMessageOfNull(nameof(param.CultureInfo)), nameof(param)); }
-            if (param.CustomerId == Guid.Empty) { throw new ArgumentException(GetMessageOfEmpty(nameof(param.CustomerId)), nameof(param)); }
-            if (string.IsNullOrWhiteSpace(param.Scope)) { throw new ArgumentException(GetMessageOfNullWhiteSpace(nameof(param.Scope)), nameof(param)); }
+            if (param == null) throw new ArgumentNullException(nameof(param));
+            if (param.CultureInfo == null) throw new ArgumentException(GetMessageOfNull(nameof(param.CultureInfo)), nameof(param));
+            if (param.CustomerId == Guid.Empty) throw new ArgumentException(GetMessageOfEmpty(nameof(param.CustomerId)), nameof(param));
+            if (string.IsNullOrWhiteSpace(param.Scope)) throw new ArgumentException(GetMessageOfNullWhiteSpace(nameof(param.Scope)), nameof(param));
 
             var request = new GetCartsByCustomerIdRequest()
             {
                 CustomerId = param.CustomerId,
                 ScopeId = param.Scope,
                 CultureName = param.CultureInfo.Name,
-                CartType = CartConfiguration.RecurringOrderCartType
```

Comment:
Method says GetRecurringCartsAsync - so I think need to leave here  CartType = CartConfiguration.RecurringOrderCartType